exersices/Ex04/myrand.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Sindre Tallaksrud'
+__email__ = 'sindrtal@nmbu.no'
+
+
+class LCGRand:
+    def __init__(self, seed):
+        self.seed = seed
+        self.count = 0
+        self.r = [0]
+
+    def rand(self):
+        self.r[0] = self.seed
+        self.r.append(0)
+        a = 7**5
+        m = 2**31-1
```

Comment:
Instead of recalculating these numbers everytime `rand()` is called, you can put it before the `__init__` function (but still within this class) and reuse it everytime `rand()` is called